src/modules/crypto.js

```diff
@@ -35,15 +38,15 @@ class Crypto {
         }
     }
 
-    async verifyMultihash(pubKeyMultihash, multihash, signature) {
+    async verifyMultihash(pubKeyMultihash, multihash, signature, pubKeyValue = null) {
```

Comment:
Used by the OrbitDB Index which now gets "resolved" review records so it already has the public key, don't have to invoke IPFS again